INSTALL.txt

```diff
@@ -106,8 +106,8 @@ password, database name).
     - Please check the additional notes if running on MySQ 5.7.17 or newer
 
 4. If you know in advance that it will be impossible for the web server to
-write or create files in your web directory, create an empty file called
-.htconfig.php and make it writable by the web server.
+write or create files in the /config/ subfolder, create an empty file called
```

Comment:
Is the `/` in front of the path intended? Why would we need a directory at the base of the directory hierarchy? And should it then not be something like `/etc/friendica/` or something similar?

Reply:
Not it isn't. It is a subfolder because we can't presume the file structure of the hosting server. `/etc/friendica/` wouldn't work on Windows.

Reply:
So, can you remove the leading `/` then`It suggests that you mean a directory in the root of the file system.